### PR TITLE
perf(wpf): Replace Reflection with PInovke win32 in WpfKeyboardInputSource

### PR DESCRIPTION
**Pull request by lindexi**
## PR Type

What kind of change does this PR introduce?

- Refactoring

## What is the current behavior?

We have to do Reflection on every keyboard event in wpf.

## What is the new behavior?

We will call the win32 method the get the scan code.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features) no need
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable) 
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Reference https://github.com/unoplatform/uno/pull/14371#discussion_r1404625891 and https://github.com/dotnet/wpf/issues/8473#issuecomment-2084878655

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs [hunk 1]
```diff
@@ -156,8 +136,11 @@ private static VirtualKeyModifiers GetKeyModifiers(ModifierKeys modifierKeys)
 		return modifiers;
 	}
 
-	private VirtualKey ConvertKey(Key key)
+	private static VirtualKey ConvertKey(Key key)
 	{
 		return (VirtualKey)System.Windows.Input.KeyInterop.VirtualKeyFromKey(key);
 	}
+
+	[DllImport("User32.dll")]
```
Comment (Youssef1313) [inline src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs:144]: Could this be LibraryImport?

Comment (lindexi) [reply src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs:144]: @Youssef1313  I find your PR (https://github.com/unoplatform/uno/pull/16556) will replace all DllImport with LibraryImport. But I find the `InputHelper.MapVirtualKey` still use DllImport. Should I use LibraryImport here?

Comment (Youssef1313) [reply src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs:144]: My PR was focused on X11, but can do a follow-up everywhere in the codebase. I'm okay with it being DllImport. We can change later if needed.

**Review by Youssef1313 — APPROVED** [review]

File: src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs (hunk 1 above)
Comment (lindexi) [reply src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs:144]: Thank you @Youssef1313

**Comment by Youssef1313** [comment]
@lindexi Thank you for the contribution!

**Comment by lindexi** [comment]
Thank you.

**Pull request merged**